prow/jobs/ti-community-infra/lifecycle-periodics.yaml

```diff
@@ -0,0 +1,109 @@
+              Reopen the issue with `/reopen`.
+              Mark the issue as fresh with `/remove-lifecycle rotten`.
+              Send feedback to sig-community-infra or [hi-rustin](https://github.com/hi-rustin).
+              /close
+            - --template
+            - --ceiling=10
+            - --confirm
+          volumeMounts:
+            - name: github-token
+              mountPath: /etc/github
+              readOnly: true
+      volumes:
+        - name: github-token
+          secret:
+            secretName: github-token
+
+  - name: periodic-ti-community-infra-rotten
+    interval: 1h
+    decorate: true
+    annotations:
+      description: Adds lifecycle/rotten to stale issues after 30d of inactivity
+    spec:
+      containers:
+        - image: gcr.io/k8s-prow/commenter:v20201217-2de500de27
+          command:
+            - /app/robots/commenter/app.binary
+          args:
+            - |-
+              --query=org:ti-community-infra
+              -label:lifecycle/frozen
+              label:lifecycle/stale
+              -label:lifecycle/rotten
+            - --updated=720h
+            - --token=/etc/github/token
+            - |-
+              --comment=Stale issues rot after 30d of inactivity.
+              Mark the issue as fresh with `/remove-lifecycle rotten`.
+              Rotten issues close after an additional 30d of inactivity.
+              If this issue is safe to close now please do so with `/close`.
+              Send feedback to sig-community-infra or [hi-rustin](https://github.com/hi-rustin).
+              /lifecycle rotten
+            - --template
+            - --ceiling=10
+            - --confirm
+          volumeMounts:
+            - name: github-token
+              mountPath: /etc/github
+              readOnly: true
+      volumes:
+        - name: github-token
+          secret:
+            secretName: github-token
+
+  - name: periodic-ti-community-infra-stale
+    interval: 1h
+    decorate: true
+    annotations:
+      description: Adds lifecycle/stale to issues after 30d of inactivity
+    spec:
+      containers:
+        - image: gcr.io/k8s-prow/commenter:v20201217-2de500de27
+          command:
+            - /app/robots/commenter/app.binary
+          args:
+            - |-
+              --query=org:ti-community-infra
+              -label:lifecycle/frozen
+              -label:lifecycle/stale
+              -label:lifecycle/rotten
+            - --updated=2160h
+            - --token=/etc/github/token
+            - |-
+              --comment=Issues go stale after 90d of inactivity.
+              Mark the issue as fresh with `/remove-lifecycle stale`.
+              Stale issues rot after an additional 30d of inactivity and eventually close.
+              If this issue is safe to close now please do so with `/close`.
+              Send feedback to sig-community-infra or [hi-rustin](https://github.com/hi-rustin).
+              /lifecycle stale
```

Comment:
I think it's right.

Reply:
The description is `Adds lifecycle/stale to issues after 30d of inactivity`， but the comment is `Issues go stale after 90d of inactivity.`.
So how many days does it take to be inactive for an issue to become stale?

Reply:
Good catch! Fixed.